doozer/doozerlib/cli/release_gen_payload.py

```diff
@@ -761,25 +782,44 @@ async def sync_payloads(self):
             False: dict()
         }
 
-        # Ensure that all payload images have been mirrored before updating
-        # the imagestream. Otherwise, the imagestream will fail to import the
-        # image.
+        # Check to see if there is an embargoed build and whether --embargo-permit-ack is specified
+        filtered_payload_entries_for_arch = {}
+        for arch, payload_entries in self.payload_entries_for_arch.items():
+            filtered_payload_entries = {}
+            for payload_tag_name, payload_entry in payload_entries.items():
+                if payload_entry.rhcos_build is not None:
+                    # RHCOS builds can't be embargoes for now
+                    # Filter them out
+                    continue
+
+                if self.runtime.assembly_type is AssemblyTypes.STREAM:
```

Comment:
think you only wanted to filter embargoed ones here... this filters everything for `STREAM`